README.md

```diff
@@ -114,8 +101,8 @@ Send your gateway Ligtning node some funds:
 # Get address from gateway lightning node (we'll call it LN_ADDRESS)
 lightning-cli --network regtest --lightning-dir=ln1 newaddr
 
-# Send bitcoin to gateway lightning address
-bitcoin-cli -regtest -rpcuser=bitcoin -rpcpassword=bitcoin sendtoaddress <LN_ADDRESS> <AMOUNT>
+# Send bitcoin to gateway lightning address, and mine a block to confirm it
+bitcoin-cli -regtest -rpcuser=bitcoin -rpcpassword=bitcoin sendtoaddress <LN_ADDRESS> 1.0
```

Comment:
Should have large enough denominations for this to not take days (e.g. if the server had to issue 100'000 tokens)